woodwork/utils.py

```diff
@@ -543,3 +539,12 @@ def _replace_nan_strings(df):
         df[col] = replaced_series
 
     return df
+
+
+def _is_tuple_nan(value):
```

Comment:
Not sure I'd expect `_is_tuple_nan(np.nan)` to be True based off the name. There's a somewhat similar util `_is_null_latlong`, but that just focuses on individual values within the tuple and not the tuple iteslf. It does make me wonder if somehow they could be combined or both used for this purpose.

Reply:
@tamargrey good catch. I think I should add another condition there to see if its a tuple